packages/common/src/ui/layout/tables/components/DataRow/DataRow.tsx

```diff
@@ -56,6 +68,7 @@ export const DataRow = <T extends RecordWithId>({
           }}
           onClick={onRowClick}
           hover={hasOnClick}
+          selected={isFocused}
```

Comment:
Using the Mui `selected` state to show focus, which is kind of confusing since we have a separated "selected" state using the checkboxes. It should really be the same as the "hover" state. Is there a simple way to "fake" the hover state so it automatically uses the Mui hover styling?

Reply:
I don't know of a simple way. Only to drop down into the browser apis, get the dom node and set the hover state 💩 

Yes using the selected state is a little confusing, but I do like the colour differentiation between your focus and hovering (and also selected..). 

I think it's looks great. Maybe if you want to avoid using the selected state (and use that for the actual selected state..), then you could just apply a background to the table row directly.. or, use the `rowStyle` in the table context 🤔

Reply:
Okay have done it like this: ad2d979b
```
            backgroundColor: isFocused ? 'background.menu' : null,
```

Have also updated the default hover background colour to use a palette value (rather than MUI default). I like having the 'focused' and 'hover' look the same, since they're basically indicating the same thing -- one with the keyboard, one with the mouse.